### PR TITLE
S tuli/updatenetcore

### PR DESCRIPTION
**Pull request by s-tuli**
Updating AspNetCore Nugets to 2.2
Testing: Run various Apis using postman

### CODE REVIEW

File: Services/DataX.ServiceHost/DataX.ServiceHost/DataX.ServiceHost.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
```
Comment (kjcho-msft) [inline Services/DataX.ServiceHost/DataX.ServiceHost/DataX.ServiceHost.csproj]: please change this back to true

File: Services/DataX.Gateway/DataX.Gateway/ApplicationParameters/Cloud.xml [hunk 2]
```diff
@@ -9,15 +9,15 @@ Licensed under the MIT License
     <Parameter Name="Auth_MinReplicaSetSize" Value="2" />
     <Parameter Name="Auth_TargetReplicaSetSize" Value="3" />
     <Parameter Name="Api_InstanceCount" Value="-1" />
-    <Parameter Name="SSL_Cert_Thumbprint" Value="" />
-    <Parameter Name="ClientId" Value="" />
-    <Parameter Name="Audience" Value="" />
-    <Parameter Name="ReverseProxySslThumbprint" Value="" />
-    <Parameter Name="AllowedUserRoles" Value="" />
+    <Parameter Name="SSL_Cert_Thumbprint" Value="62BEBAE76C57023F7B0B807F9C8836A12FB530DA" />
```
Comment (kjcho-msft) [inline Services/DataX.Gateway/DataX.Gateway/ApplicationParameters/Cloud.xml]: Please remove all gateway changes.

File: Services/DataX.sln [hunk 3]
```diff
@@ -1,549 +1,559 @@
...
-		{9E338FCC-793F-4048-9672-048270C856C4} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{D5D3DD2D-760A-4818-A74F-C0061719841C} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {FB461262-8931-453E-8214-506BF892338E}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```
Comment (kjcho-msft) [inline Services/DataX.sln]: Do we need this change? if so, why?

File: Services/DataX.ServiceHost/DataX.ServiceHost/DataX.ServiceHost.csproj (hunk 1 above)
Comment (s-tuli) [reply Services/DataX.ServiceHost/DataX.ServiceHost/DataX.ServiceHost.csproj]: done

File: Services/DataX.sln (hunk 3 above)
Comment (s-tuli) [reply Services/DataX.sln]: This was always 15. We can make a pass for all of the solutions in our repo and move them all to 16. For now, for consistency, wil leave this as 15.

File: Services/DataX.Gateway/DataX.Gateway/ApplicationParameters/Cloud.xml (hunk 2 above)
Comment (s-tuli) [reply Services/DataX.Gateway/DataX.Gateway/ApplicationParameters/Cloud.xml]: done

**Pull request merged**